src/components/article/slideshow/slideshow.js

```diff
@@ -118,31 +121,32 @@ export class Slideshow extends Component {
     let thumbnailsWidth = this.state.slideshowWidth - thumbnailsPadding * 2
 
     return (
-      <div
-        className={styles['ss-slider']}
-        ref={i => this._slideshow = i}
-      >
+      <LazyLoad offsetTop={UI_SETTING.image.loadingOffset.placeholder}>
```

Comment:
you could set the offsetTop property to LazyLoad like:
offsetTop={UI_SETTING.image.loadingOffset.placeholder} ...